test/modules/standard/jama/TestMatrix.chpl

```diff
@@ -0,0 +1,403 @@
+use Jama;
```

Comment:
You mean the Jama source?  I'd be fine with it directly under modules/standard.  The test code is probably best placed in a new folder test/modules/standard/LinearAlgebraJama